jupyter_ydoc/ybasedoc.py

```diff
@@ -31,6 +31,9 @@ def __init__(self, ydoc: Optional[Doc] = None):
             self._ydoc = Doc()
         else:
             self._ydoc = ydoc
+
+        self._awareness = awareness
```

Comment:
I think we should just have:
```suggestion
        self.awareness = awareness
```
And remove the getter and setter as they do nothing.

Reply:
Or maybe we should only remove the setter, to make `awareness` read only, it is not supposed to be set later.
What do you think ?

Reply:
I removed the getter and setter.